packages/app-store-cli/src/build.ts

```diff
@@ -13,25 +13,17 @@ import type { AppMeta } from "@calcom/types/App";
 import { APP_STORE_PATH } from "./constants";
 import { getAppName } from "./utils/getAppName";
 
-let isInWatchMode = false;
-if (process.argv[2] === "--watch") {
-  isInWatchMode = true;
-}
+const isInWatchMode = process.argv[2] === "--watch";
 
 const formatOutput = (source: string) =>
   prettier.format(source, {
     parser: "babel",
     ...prettierConfig,
   });
 
-const getVariableName = function (appName: string) {
-  return appName.replace(/[-.]/g, "_");
-};
+const getVariableName = (appName: string) => appName.replace(/[-.]/g, "_");
 
-const getAppId = function (app: { name: string }) {
-  // Handle stripe separately as it's an old app with different dirName than slug/appId
```

Comment:
While comments can be stale and/or not that useful, this one actually is. We should keep it in. Let's just add an 
// INFO: Handle stripe separately as it's an old app with different dirName than slug/appId above the function definition.

Reply:
sounds right, just did that